pom.xml

```diff
@@ -158,12 +158,18 @@
             <dependency>
                 <groupId>io.opentracing</groupId>
                 <artifactId>opentracing-api</artifactId>
-                <version>0.30.0</version>
+                <version>0.31.0</version>
             </dependency>
             <dependency>
                 <groupId>com.uber.jaeger</groupId>
                 <artifactId>jaeger-core</artifactId>
-                <version>0.21.0</version>
+                <version>0.23.0</version>
```

Comment:
we should consider moving this into tchannel-crossdock module, rather than claiming it a dependency of the whole tchannel